templates/hotgraphicLayoutTiles.jsx

```diff
@@ -0,0 +1,52 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { classes, compile } from 'core/js/reactHelpers';
+
+export default function HotgraphicLayoutTiles(props) {
+  const globals = Adapt.course.get('_globals');
+  const visitedLabel = globals?._accessibility?._ariaLabels.visited;
+
+  const {
+    _items,
+    onPinClicked
+  } = props;
+
+  return (
+    <div className="hotgraphic__tile-item-container" role="list">
+
+      {_items.map(({ _index, _graphic, _isVisited, title }) => {
+
+        const visited = _isVisited ? visitedLabel + '. ' : '';
+        const itemTitle = title + '. ';
+        const itemCount = compile(globals._components._hotgraphic.item, { itemNumber: _index + 1, totalItems: _items.length });
+        const ariaLabel = visited + itemTitle + itemCount;
```

Comment:
@joe-allen-89 Yeah, my thought was that it's is easier to see that it's three separate "things" if they're split up. One long concatenated string is somewhat difficult to process in my mind. Happy to change if everyone feels differently.